data/data/bootstrap/systemd/units/release-image.service.template

```diff
@@ -6,4 +6,7 @@ After=network-online.target
 [Service]
 Type=oneshot
 ExecStart=/usr/local/bin/release-image-download.sh
+{{if .IsOKD -}}
+ExecStart=/usr/local/bin/bootstrap-pivot.sh
```

Comment:
We now have events captured for each service in a json file for the service. This is done by sourcing `/usr/local/bin/bootstrap-service-record.sh` at the beginning of the script run for the service. If we add a second `ExecStart` for the service, this will make the events file confusing as it will report that the service ran multiple times. Could we instead have one `ExecStart` that runs a script that then runs the release-image-download and bootstrap-pivot scripts?

Reply:
Good idea, implemeted those in 0de397cd0